extensions/amp-facebook/0.1/amp-facebook.js

```diff
@@ -41,11 +41,10 @@ class AmpFacebook extends AMP.BaseElement {
     const iframe = getIframe(this.element.ownerDocument.defaultView,
         this.element, 'facebook');
     this.applyFillContent(iframe);
+    const amp = this.element;
     // Triggered by context.updateDimensions() inside the iframe.
     listenFor(iframe, 'embed-size', data => {
-      iframe.height = data.height;
-      iframe.width = data.width;
-      const amp = iframe.parentElement;
+      // TODO REVIEW
```

Comment:
We need to drop width and height from iframe, but the actual width/height are not changed by them given the "fill" CSS. 

It appears that they already use `changeHeight`. I believe this component got an exemption at the same time as Twitter widget.